src/psyclone/tests/domain/common/algorithm/alg_psyir_test.py

```diff
@@ -526,14 +563,105 @@ def test_aic_lowertolanguagelevel_multi_invoke():
 
     assert len(psyir.walk(AlgorithmInvokeCall)) == 0
     assert len(psyir.walk(KernelFunctor)) == 0
-
     call1 = psyir.children[0][0]
+    with pytest.raises(KeyError) as info:
```

Comment:
Could this just be `assert "invoke" not in call1.scope.symbol_table._symbols` - it feels a bit like we're testing the `lookup` method otherwise? I feel the same about the check at L475.

Reply:
Good call. I wrongly thought that `_symbols` returned symbols not strings and concluded that lookup was therefore simplest.